spec/requests/feed_requests_spec.rb

```diff
@@ -36,6 +36,17 @@
       expect(first_entry["id"]).to eq(id)
       expect(first_entry["updated"]).to eq(updated)
     end
+
+    it "uses created_at instead of updated_at for updated value" do
```

Comment:
Sorry that sounded mean. I just was lazy with capitalization but not with punctuation.